impress-org/givewp
# Feature: add initial banner

Resolves: [GIVE-870] & [GIVE-871]

## Description
This PR includes the initial markup required to render a single StellarSaleBanner. The banner should display on the Donor, Donation & Reports pages. If the banner is closed it should remain closed on all 3 pages.

### Additional details:
- An update to the sales-banner.js file was made to ensure the banner for the reports page is displayed in the same place as the others.
- An improvement of the StellarSalesBanner stylesheet which include updates with the styling & file location.
- The links for the banner are placeholders for now.

## Affects
Donation, Donor, Reports pages

## Visuals

https://github.com/impress-org/givewp/assets/75056371/32223671-d209-4d0f-8173-fe3ea4ea2dc5

## Testing Instructions
- adjust the date-range in the StellarSaleBanner class inside the `getBanners` method, this should enable the banner.
- visit the Donation, Donor & Report pages to verify the banner is displayed.
- close the banner by clicking the icon in the top right and verify it remains closed on all 3 pages.

## Pre-review Checklist




[GIVE-870]: https://stellarwp.atlassian.net/browse/GIVE-870?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[GIVE-871]: https://stellarwp.atlassian.net/browse/GIVE-871?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
